ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cssobj", 
+  "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names, media query hook for IE8, styled components",
+  "keywords": [
+    "cssom",
+    "prefixer",
+    "css generate",
+    "diff css",
+    "media query",
+    "css modules"
+  ],
+  "author": "James Yang <jamesyang999@gmail.com>", 
+  "homepage": "https://cssobj.github.io/cssobj-demo/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cssobj/cssobj.git"
+  },
+  "filename": "cssobj.js",
+  "npmName": "cssobj",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/cssobj.!(amd|cjs|es|iife)*"
```

Comment:
As your comment, let's modify to `"**/cssobj.!(amd|cjs|es)*"`
This will add `cssobj.min.js`, `cssobj.umd.js`, `cssobj.iife.js` files.
Thanks. :)